### ISSUE TITLE
Is it possible to add convolution filters to 3D MPR Viewer ?

### ISSUE THREAD

**Issue by mcConti** [horosproject/horos#341]
Is it possible to add "convolution filters" to 3D MPR Viewer ?
Especially "sharpen 3x3" filter.
thx

*************
Is there any newer build than the official 3.0.1?
thx

**Comment by omrd** [comment]
I agree with this feature request. Adding filters to the MPR views (curved and non-curved) would be a great addition. 

@mcConti you can add convolution filters on the 2D view, apply the filters to the raw data and then open in MPR. It's annoying but it works.

**Comment by air-h-128k-il** [comment]
you wrote:
Is there any newer build than the official 3.0.1?

Yes. My version of Horos ( https://github.com/air-h-128k-il/HorliX ) is slightly different from official Horos. Both are almost same in source code but diffrent in libralies.
Because I re-compiled libralies under 64bit mode using Xcode 9.4.

**Comment by brizolara** [comment]
@mcConti Later today a new release candidate will be available at the download page.

Nice workaround, @omrd's! Then, to get back to the unfiltered images, one have to go to the default viewer window and go to menu 2D Viewer->Revert Series.

Anyway, I'm checking the viability of getting the filters to work on the MPRs.

Thank you for the nice observations!
